<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureSystem</class>
 <widget class="QWidget" name="ConfigureSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="group_system_settings">
       <property name="title">
        <string>System Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_language">
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_birthday">
          <property name="text">
           <string>Birthday</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_console_id">
          <property name="text">
           <string>Console ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_birthday2">
          <item>
           <widget class="QComboBox" name="combo_birthmonth">
            <item>
             <property name="text">
              <string>January</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>February</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>March</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>April</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>May</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>June</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>July</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>August</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>September</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>October</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>November</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>December</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_birthday"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combo_language">
          <property name="toolTip">
           <string>Note: this can be overridden when region setting is auto-select</string>
          </property>
          <item>
           <property name="text">
            <string>Japanese (日本語)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>French (français)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>German (Deutsch)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Italian (italiano)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spanish (español)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chinese</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Korean (한국어)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dutch (Nederlands)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Portuguese (português)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Russian (Русский)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Taiwanese</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>British English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Canadian French</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Latin American Spanish</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Simplified Chinese</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Traditional Chinese (正體中文)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_sound">
          <property name="text">
           <string>Sound output mode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_sound">
          <item>
           <property name="text">
            <string>Mono</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stereo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Surround</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="button_regenerate_console_id">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Regenerate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gridGroupBox">
       <property name="title">
        <string>Profile Manager</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Current User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="current_user_icon">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="interactive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="current_user_username">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pm_set_image">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Set Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pm_add">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pm_rename">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Rename</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pm_remove">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_disable_info">
       <property name="text">
        <string>System settings are available only when game is not running.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
